src/PowerShellEditorServices/Workspace/Workspace.cs

```diff
@@ -624,6 +624,65 @@ private static string UnescapeDriveColon(string fileUri)
             return sb.ToString();
         }
 
+        /// <summary>
+        /// Converts a file system path into a DocumentUri required by Language Server Protocol.
+        /// </summary>
+        /// <remarks>
+        /// When sending a document path to a LSP client, the path must be provided as a
+        /// DocumentUri in order to features like the Problems window or peek definition
+        /// to be able to open the specified file.
+        /// </remarks>
+        /// <param name="path">
+        /// A file system path. Note: if the path is already a DocumentUri, it will be returned unmodified.
+        /// </param>
+        /// <returns>The file system path encoded as a DocumentUri.</returns>
+        public static string ConvertPathToDocumentUri(string path)
+        {
+            const string fileUriPrefix = "file:///";
+
+            if (path.StartsWith("untitled:", StringComparison.Ordinal))
```

Comment:
you have `fileUriPrefix` in its own variable, but `untitled:` is used directly.

Maybe move it out to a 

```csharp
const string untitledUriPrefix = "untitled:";
```

Reply:
We had this in a variable because we *were* using that string in multiple places. That is no longer the case, so I think I'll remove the `fileUriPrefix` variable and use the string in the one place it is needed.

Reply:
Turns out we still use that var twice so I'll create a var for untitled.